src/fundus/publishers/us/reuters.py

```diff
@@ -59,3 +61,18 @@ def topics(self) -> List[str]:
             # Remove empty topics and duplicates deterministically
             processed_topics = list(dict.fromkeys(topic for topic in topics if topic))
             return processed_topics
+
+    class V1_1(V1):
```

Comment:
There are some cases of subheadlines: e.g. https://www.reuters.com/world/us/takeaways-state-union-address-2024-03-08/ or https://www.reuters.com/markets/global-markets-wrapup-1-2024-03-08/

Reply:
Nice you found this 👍 I added the selector to the original version as well.